src/http/requests/chart.request.ts

```diff
@@ -18,4 +18,11 @@ export class ChartRequest {
     @IsNotEmpty()
     @IsString()
     type: ChartTypes;
+
+    @ApiProperty()
+    @IsOptional()
+    @IsNotEmpty()
+    @IsString()
+    @IsIn([ChartGroupType.GROUP_DAILY, ChartGroupType.GROUP_MONTHLY, ChartGroupType.GROUP_YEARLY])
+    group_type: ChartGroupType;
```

Comment:
If it's flagged as optional then the object property itself must be optional with `?`